src/vsr/replica.zig

```diff
@@ -9308,8 +9330,21 @@ pub fn ReplicaType(
 
         fn sync_reclaim_tables(self: *Replica) void {
             while (self.grid.blocks_missing.reclaim_table()) |queue_table| {
+                log.info(
+                    "sync_reclaim_tables: table synced: address={} checksum={} wrote={}/{?}",
+                    .{
+                        queue_table.index_address,
+                        queue_table.index_checksum,
+                        queue_table.table_blocks_written,
+                        queue_table.table_blocks_total,
+                    },
+                );
+
                 const table: *RepairTable = @fieldParentPtr("table", queue_table);
                 self.grid_repair_tables.release(table);
+                self.trace.stop(.{ .replica_sync_table = .{
```

Comment:
Do we need reset for the case where we jump from sync to sync?

Reply:
No, in that case we call `GridBlocksMissing.cancel()`:

```zig
    pub fn cancel(queue: *GridBlocksMissing) void {
        assert(queue.checkpointing == null);

        queue.faulty_blocks.clearRetainingCapacity();
        while (queue.faulty_tables.pop()) |table| {
            queue.faulty_tables_free.push(table);
        }

        queue.* = .{
            .options = queue.options,
            .faulty_blocks = queue.faulty_blocks,
            .faulty_tables_free = queue.faulty_tables_free,
        };
    }
```

which still results in the table being reclaimed.